src/lmic/radio.c

```diff
@@ -485,7 +485,7 @@ static void txfsk () {
     writeBuf(RegFifo, LMIC.frame, LMIC.dataLen);
 
     // enable antenna switch for TX
-    hal_pin_rxtx(1);
+    hal_pin_rxtx(LMIC_ANTENNA_SWITCH_TX);
```

Comment:
If we move the polarity to the wiring, we don't need this change. Instead we'd change `hal_pin_rxtx()`